content/docs/command-reference/commit.md

```diff
@@ -279,5 +279,5 @@ $ dvc status
 Data and pipelines are up to date.
 ```
 
-Nothing special is required, we simply `commit` to both the SCM and DVC. Since
+Nothing special is required, we simply `commit` to both the Git and DVC. Since
```

Comment:
This change is correct but can you also please improve the sentence? It's not greatly written "to both the Git and DVC" doesn't sound great.

Reply:
Okay. Will do that.